libs/core/state/src/states/state.enum.ts

```diff
@@ -0,0 +1,58 @@
+/**
+ * The DaffState enumerates the vast majority of state modifications that can exist 
+ * in UI applications. 
+ */
+export enum DaffStates {
+  /**
+   * The "Stable" state represents when no action is being taken upon an object.
+   */
+  Stable = 'Stable',
+
+  /**
+   * The "Resolving" state represents when there's uncertainty in the current status
```

Comment:
Does it represent uncertainty? I would say it's explicitly just for the immutable loading of some data, analogous to a classic GET request.

Reply:
I think we need to highlight that we're typically doing this when we don't already know the information. E.g. We're uncertain whether or not the content we're attempting to load may or may not exist.